src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -0,0 +1,88 @@
+import { View } from 'marionette';
+import hbs from 'handlebars-inline-precompile';
+
+import 'scss/modules/buttons.scss';
+import 'scss/modules/sidebar.scss';
+
+import { animSidebar } from 'js/anim';
+
+import Droplist from 'js/components/droplist';
+
+import FiltersSidebarTemplate from './filters-sidebar.hbs';
+
+import './filters-sidebar.scss';
+
+const SidebarGroupsDropList = Droplist.extend({
+  popWidth() {
+    return this.getView().$el.outerWidth();
+  },
+  viewOptions: {
+    className: 'button-secondary--compact w-100',
```

Comment:
I don't think this should be `--compact`

Reply:
You are correct. Removed `--compact` in last push.